aten/src/ATen/native/nested/NestedTensorUtils.cpp

```diff
@@ -167,5 +170,36 @@ std::vector<Tensor> split_with_sizes_nested(
   return splits;
 }
 
+Tensor get_nested_sizes_from_sym_sizes(const c10::SymIntArrayRef& size) {
+  if (size.size() == 1) {
+    auto out = at::ones({}, TensorOptions().dtype(at::kLong));
+    return out;
```

Comment:
Are you sure you want a scalar here? TBH I'm not really sure what this function does

Reply:
The sym_size function goes from column 1 to column 2, and this function goes from column 2 back to column 1. (You were right - previous iteration had missed a case.)

```
                              nested size           shape
NT with 0 constituents        torch.tensor(1)       Size([0])
NT with scalar constituents   torch.empty([N, 0])   Size([N])
NT with K-d constituents      N x K tensor          Size([N, j0, j1, ..., jk])
```